src/ParallelAlgorithms/Amr/Projectors/TimeStepping.hpp

```diff
@@ -0,0 +1,121 @@
+// Distributed under the MIT License.
```

Comment:
Can we move this to `Time/` or `Evolution/`? As discussed in a call recently I think we have to be careful not to repeat the mistake we made with the parallel interpolation framework, which couples most of the code together. Libs should depend on AMR and implement their own projectors/mutators, not the other way round. I've had good success implementing AMR mutators right next to each initialization action, even in the same file. That worked well because usually you have to project all the same tags during AMR that you initialize at the beginning. When the AMR and init mutators live next to each other then they can be maintained together, possibly even sharing functions.